tasks/sync-granule/GranuleFetcher.js

```diff
@@ -338,14 +340,15 @@ class GranuleFetcher {
    * @param {Object} file - file to download
    * @param {string} destinationBucket - bucket to put file in
    * @param {string} duplicateHandling - how to handle duplicate files
+   * @param {string | undefined} ACL
```

Comment:
```suggestion
   * @param {string} [ACL]
```

Reply:
Given the types in the files up above, this should be `ObjectCannedACL`, but that opens a can of worms that isn't worth getting into in this PR.